### PR TITLE
[8.0] Port account_fiscal_position_rule_purchase module

### PR DESCRIPTION
**Pull request by Ehtaga**

### CODE REVIEW

File: account_fiscal_position_rule_purchase/models/purchase_order.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+class PurchaseOrder(models.Model):
+    _inherit = 'purchase.order'
+
+    @api.model
+    def _fiscal_position_map(self, result, company_id, partner_id,
+                             partner_invoice_id, partner_shipping_id,
+                             ):
+        fp_rule_obj = self.env['account.fiscal.position.rule'].with_context(
+            {'use_domain': ('use_purchase', '=', True)}
```
Comment (bwrsandman) [inline account_fiscal_position_rule_purchase/models/purchase_order.py]: better:

``` python
        fp_rule_obj = self.env['account.fiscal.position.rule'].with_context(
            use_domain=('use_purchase', '=', True)
        )
```

File: account_fiscal_position_rule_purchase/README.rst [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+* ...
+
+For further information, please visit:
+
+* https://www.odoo.com/forum/help-1
+
+Known issues / Roadmap
+======================
+
+* ...
```
Comment (bwrsandman) [inline account_fiscal_position_rule_purchase/README.rst:31]: Remove the empty sections

**Comment by danimaribeiro** [comment]
This is working as expected. :+1: , the onchange changes the fiscal position correctly.
Just one question about the behavior when is a purchase:

In the rules are two groups (origin and destination).
When is a sale, origin is the company, and destination the customer, but in the case of a purchase?
Today the origin continues to be the company, and the destination is the supplier.

**Comment by mileo** [comment]
LGTM, Tested:
- Fiscal postion rule;
- Mapping tax of purchase with #28 
  :100:

**Comment by rvalyi** [comment]
sorry but superseded by anterior PR https://github.com/OCA/account-fiscal-rule/pull/11 now merged

**Pull request closed without merging**